book/src/assets/how_to_use_assets.md

```diff
@@ -0,0 +1,221 @@
+# How to Use Assets
+
+This guide covers the basic usage of assets into Amethyst for existing supported formats. For a list of supported formats, please [use this search for "Format"][doc_search_format] in the API documentation, and filter by the following crates:
+
+* amethyst_assets
+* amethyst_audio
+* amethyst_gltf
+* amethyst_locale
+* amethyst_ui
+
+## Steps
+
+1. Instantiate the Amethyst application with the assets directory.
+
+    ```rust,no_run,noplaypen
```

Comment:
Is there a reason to no_run these examples? They'd make for helpful doctests.

Reply:
1. could be uncommented, but doesn't really do much.
2. uses `RenderBundle` which would fail on headless CI
3. would attempt to load the non-existent asset `"player.png"`, which wouldn't hard error, but may be noisy.
4. would wait for a progress tracker that never ends

I can uncomment 1. and 3., but i think the value isn't worth the cost

Reply:
It's fine for this PR I think but I'd really like if we looked into making all the examples in the book runnable for doctests with mocks for things like RenderBundle.